languages/csharp/exercises/concept/classes/.docs/introduction.md

```diff
@@ -0,0 +1,66 @@
+The primary object-oriented construct in C# is the _class_, which is a grouping of related data (_fields_) and behavior (_methods_). The fields and methods of a class are known as its _members_.
+
+Access to members can be restricted through access modifiers, the two most common ones being:
+
+- `public`: the member can be accessed by any code (no restrictions).
+- `private`: the member can only be accessed by code in the same class.
+
+You can think of a class as a template for creating instances of that class. To create an instance of a class (also known as an _object_), the `new` keyword is used:
+
+```csharp
+class Car
+{
+}
+
+// Create two car instances
+var myCar = new Car();
+var yourCar = new Car();
+```
+
+Fields have a type and a name (defined in camelCase) and can be defined anywhere in a class:
```

Comment:
Should the casing for the class name be mentioned somewhere as well? Especially as it is different.

Reply:
Good point. I've added it to this sentence to have both naming conventions near to each other.